packages/unpublished-relationship-performance-test-app/app/routes/application.js

```diff
@@ -19,4 +19,22 @@ export default Route.extend({
       return flattened;
     });
   },
+  afterModel() {
```

Comment:
surprising we picked this up but seems fine it makes the perf check more backwards compatible

Reply:
https://github.com/emberjs/data/pull/6985/files#diff-3f38ec8323b0ede1b00d2042d47df6f3